lib/views/home_page.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+import 'package:hw_2/app/app_header.dart';
+import 'package:hw_2/app/app_body.dart';
+import 'package:hw_2/app/app_footer.dart';
+import 'package:hw_2/main.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  refresh() {
+    setState(() {});

```

Comment:
this is a sign of hack, I mentioned above: this widget actually doesn't have mutable state, but you are using `StatefulWidget` + `setState` with empty object to trigger the re-render